nrcatalogtools/lvc.py

```diff
@@ -705,78 +716,84 @@ def get_nr_to_lal_rotation_angles(
+                print("Choosing available reference time")
+                interp = False
     else:
-        Interp, avail_ref_time = CheckInterpReq(H5File, TRef)
+        interp, avail_ref_time = check_interp_req(h5_file, t_ref)
+
+    if interp is False:
 
-    if Interp is False:
         # Then load default values from the NR data
         # at hard coded reference time.
 
         # Get the available reference time for book keeping
-        TRef = GetRefTimeFromMetadata(Metadata)
+        t_ref = get_ref_time_from_metadata(sim_metadata)
 
-        # Check for LAL frame in metadata.
+        # Check for LAL frame in simulation metadata.
         # RIT and GT qualify this.
         # Default attributes in case of no interpolation
-        # RefCheckInterp = CheckNRAttrs(H5File, ReqTSAttrs)
-        # RefCheckDefMeta, AbsentAttrsMeta = CheckNRAttrs(Metadata, ReqDefAttrs)
-        RefCheckDefH5, AbsentAttrsH5 = CheckNRAttrs(H5File)
+        ref_check_def_h5, absent_attrs_h5 = check_nr_attrs(h5_file)
 
-        # RefCheckDefMeta, AbsentAttrsMeta  = CheckNRAttrs(Metadata, ReqDefAttrs)
+        if ref_check_def_h5 is False:
 
-        if RefCheckDefH5 is False:
             # Then the LAL source frame information is not present in the H5 file.
             # Then this could be SXS or GT data. The LAL source frame need to be computed from
-            # the H5 File or metadata.
+            # the H5 File or simulation metadata.
 
-            # Check if LAL source frame info is present in the metadata.
-            RefCheckDefMeta, AbsentAttrsMeta = CheckNRAttrs(Metadata)
+            # Check if LAL source frame info is present in the simulation metadata.
+            ref_check_def_meta, absent_attrs_meta = check_nr_attrs(sim_metadata)
 
-            if RefCheckDefMeta is False:
+            if ref_check_def_meta is False:
                 # Then this is SXS data.
-                try:
-                    # Compute the LAL source frame from metadata
-                    RefParams = ComputeLALSourceFrameFromSXSMetadata(Metadata)
-                except Exception as ex:
-                    ex(
-                        "Insufficient information to compute the LAL source frame.",
-                        f"Missing information is {AbsentAttrsH5}.",
+
+                # Check for raw information in metadata to compute the LAL source frame.
+                ref_check_def_meta_sxs, absent_attrs_meta_sxs = check_nr_attrs(
+                    sim_metadata, req_def_attrs_sxs
+                )
+
+                if ref_check_def_meta_sxs is True:
+                    # Compute the LAL source frame from simulation metadata
+                    ref_params = compute_lal_source_frame_from_sxs_metadata(
+                        sim_metadata
+                    )
+                else:
+                    raise Exception(
+                        f"Insufficient information to compute the LAL source frame.\n Missing information is {absent_attrs_meta_sxs}."
```

Comment:
Ensure line length is less than 120 by breaking the string up.

Reply:
```
                        "Insufficient information to compute the LAL source frame.\n"
                        f" Missing information is {absent_attrs_meta_sxs}."
```